sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -191,41 +188,19 @@ export class InstitutionController extends BaseController {
     const institution = await this.institutionService.getInstituteByUserName(
       user.userName,
     );
-    const institutionUserAndCount = await this.institutionService.allUsers(
```

Comment:
I believe here also we are doing the DTO transformation in service instead of controller.

Reply:
no, it's not the DTO transformation, it's getting the users paginated list and transforming, and it's the same for both ministry and institution user, in order to avoid code duplication, I have created a common service for both. we created similar in ATBC, ref - `sources/packages/api/src/services/atbc/atbc.service.ts`. let me know, if I am  missing anything or if you are not clear